app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,21 @@
+.new-comment-block{
+  border:1px solid black;
+  border-top: 0px;
+  border-bottom: 0px;
+  border-radius: 10px;
+  padding:1%;
+  margin-bottom: 2%;
```

Comment:
margin ar procentu vērtībām nebūtu labs stils - rēķini tev atkarībā no loga lieluma būs mainīgs apgalbals, kuru izmanto kā atstarpes.

Ja grībās kaut ko tipa responsive - tad parasti izmanto CSS breakpointus un atšķirīgiem izmēru apgabaliem definē fixētus atšķirīgus lielumus.

Turpmāk centies pieturēties pie fixētām vērtībām, lai nodrošinātu viendabīgu kopējo izsaktu.